### PR TITLE
Add NetSuite::ProfilesSorter

### PR DESCRIPTION
**Pull request by cored**
We need a way to create managers before the profiles that reports to
them.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -361,6 +361,3 @@ DEPENDENCIES
   title
   unicorn
   webmock
-
-BUNDLED WITH
-   1.10.6
```
Comment (bobbytables) [inline Gemfile.lock]: I think we've mentioned this at least twice now, but _please_ run `gem update bundler`

File: app/services/net_suite/profiles_sorter.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      end
+
+      attr_reader :guid
+      def initialize(guid:)
+        @guid = guid
+      end
+
+      def <=>(other_guid)
+        guid <=> other_guid
+      end
```
Comment (bobbytables) [inline app/services/net_suite/profiles_sorter.rb]: So, the problem I see with this implementation is that uuid's are not sortable. They aren't a sequential sequence.

File: Gemfile.lock (hunk 1 above)
Comment (cored) [inline Gemfile.lock]: @bobbytables ok, updating bundler

File: app/services/net_suite/profiles_sorter.rb (hunk 2 above)
Comment (cored) [inline app/services/net_suite/profiles_sorter.rb]: @bobbytables will use proper uuid in the specs to test this out.

File: Gemfile.lock (hunk 1 above)
Comment (bobbytables) [inline Gemfile.lock]: Re-bundling should add this back in.

**Comment by bobbytables** [comment]
:shipit: :+1:

**Comment by cored** [comment]
@bobbytables Reopening this because of the revert.

**Comment by bobbytables** [comment]
Thank you. Sorry for the git history mess I made.

**Pull request merged**